styles.css

```diff
@@ -0,0 +1,83 @@
+/* Topnav CSS */
+.topnav {
+  display: flex;
+  width: inherit;
+  height: 72px;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+  font-family: "Inter", sans-serif;
+  float: left;
+  color: #172b4d;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
```

Comment:
- [ ] **[REQUIRED CHANGES]** I would also like to invite you to use `flexbox` to position the elements on the page, as this is a requirement as well for the project - and trust me, flexbox will haunt you until the end of the program if you don't master it, so you should take this opportunity to improve!

Keep in mind that for the icons alignment, you will also need to use `flexbox`!

I personally recommend [this article](https://css-tricks.com/snippets/css/a-guide-to-flexbox/), it is my to-go place when I need a refresher about flexbox.

Reply:
- [ ] **[REQUIRED CHANGES]** Hello Benjamin! In case that you are having trouble implementing the `flexbox` as required instead of the `float`, let me help you:

Change your HTML to the following:
```html
    <div class="topnav">
      <div class="nav-menu">
        <a href="#news">Portfolio</a>
        <a href="#about">About</a>
        <a href="#contact">Contact</a>
      </div>
    </div>
```

Now this is where the magic happens! Instead of adding `float` to the `.topnav a`, do the following:
```css
.topnav {
  display: flex;
  justify-content: center;
}
```

and BOOM you're done like a boss 😎